main.py

```diff
@@ -303,7 +303,7 @@ def enable_interface(self, interface_enable_id=None, dpid=None):
         if not error_list:
             log.info(f"Storing administrative state for enabled interfaces.")
             self.save_status_on_storehouse()
-            return jsonify("Operation successful"), 200
+            return jsonify("Operation successful"), 201
```

Comment:
Not sure if this should be 201.

Reply:
Consensus seems to be that 201 Created is for when a new entity is created, not updated

Reply:
Thank you, @hdiogenes. I will close this PR and related issue. Also, I opened another issue #135 to standard the update requests.